src/XdebugHandler.php

```diff
@@ -96,7 +97,7 @@ public function __construct($envPrefix)
                 $this->mode = empty($modes) ? 'off' : implode(',', $modes);
             } elseif (false !== ($mode = ini_get('xdebug.mode'))) {
                 $this->mode = getenv('XDEBUG_MODE') ?: ($mode  ?: 'off');
-                if (preg_match('/^,+$/', str_replace(' ', '', $this->mode))) {
+                if (Preg::match('/^,+$/', str_replace(' ', '', $this->mode))) {
```

Comment:
Strictly speaking lots of the Preg::match could be Regex::isMatch to ensure your if works with a boolean directly, which I thought phpstan was enforcing in its strictest level, but I see now it requires adding https://github.com/phpstan/phpstan-strict-rules - just in case you wanna go even higher than 9 ;)

Reply:
Yes, I'll give that a whirl.